app/src/test/java/org/mozilla/tv/firefox/pinnedtile/PinnedTileScreenshotStoreUnitTest.kt

```diff
@@ -25,19 +26,20 @@ import java.util.UUID
  */
 @RunWith(RobolectricTestRunner::class)
 class PinnedTileScreenshotStoreUnitTest {
-
+    private lateinit var appContext: Application
     private lateinit var uuid: UUID
 
     @Before
     fun setUp() {
-        RuntimeEnvironment.application.filesDir.listFiles().forEach { it.deleteRecursively() }
+        appContext = ApplicationProvider.getApplicationContext()
+        appContext.filesDir.listFiles().forEach { it.deleteRecursively() }
         uuid = UUID.randomUUID()
     }
 
     /** Assumes [PinnedTileScreenshotStore.getFileForUUID] works correctly. */
     @Test
     fun testSaveAsyncDoesNotOverwrite() = runBlocking {
-        val context = RuntimeEnvironment.application
+        val context = appContext
```

Comment:
Nit: I think we should prefer to use `appContext` in these tests to redeclaring the variable in each.